### PR TITLE
use JS-based Tiptap parser for generating HTML on public API

### PR DESCRIPTION
**Pull request by MrGrinst**

### CODE REVIEW

File: src/Aquifer.Public.API/OpenApi/SwaggerDocumentSettings.cs [hunk 1]
```diff
@@ -32,6 +32,8 @@ All endpoints require an API key in the `api-key` header.<br><br>
 
                                  <br><br>
                                  Example applications using the Aquifer API can be found in the <a href="https://github.com/BiblioNexusStudio/aquifer-api-samples" target="_blank">samples repository on GitHub</a>.
+
+                                 <a href="https://cdn.aquifer.bible/training/webpage-search-with-react-demo.mp4" target="_blank">React Sample Application Demo</a>
```
Comment (MrGrinst) [inline src/Aquifer.Public.API/OpenApi/SwaggerDocumentSettings.cs:36]: Snuck this in as well

File: src/Aquifer.Common/Tiptap/TiptapModel.cs [hunk 2]
```diff
@@ -17,20 +16,28 @@ public class TiptapRoot<TRootContent>
     public List<TRootContent> Content { get; set; } = [];
 }
 
-public class TiptapRootContent<TRootContent>
+public class TiptapNode<TRootContent>
 {
     public virtual string Type { get; set; } = null!;
-    public virtual TiptapAttributes? Attrs { get; set; }
+
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
```
Comment (jwinston-bn) [inline src/Aquifer.Common/Tiptap/TiptapModel.cs:23]: Are these needed since it's the default option for `JsonUtilities.DefaultDeserialize`?

Comment (MrGrinst) [reply src/Aquifer.Common/Tiptap/TiptapModel.cs:23]: I also changed some other stuff, but when I tried the first time I was getting a bunch of nulls back. I'll see if my other changes fixed that and I can undo these.

Comment (MrGrinst) [reply src/Aquifer.Common/Tiptap/TiptapModel.cs:23]: @jwinston-bn just tested and it does still include the `null` without specifying this attribute:

image

**Review by jwinston-bn — APPROVED** [review]

**Pull request merged**